docs/release-notes/release-notes-0.18.3.md

```diff
@@ -81,6 +81,10 @@ blinded path expiry.
   cause UpdateAddHTLC message with blinding point fields to not be re-forwarded
   correctly on restart.
  
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/9022) a native SQL
```

Comment:
we can probably take this out since we're actually merging this PR and not #9022 (but i would appreciate my name being left in below, since i forgot to add it as part of #9021)

Reply:
done, ptal :)

Reply:
thanks!